pkgs/development/tools/tracy/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, darwin, fetchFromGitHub, tbb, gtk2, glfw, pkgconfig, freetype}:
+stdenv.mkDerivation rec {
+  name    = "tracy-${version}";
+  version = "0.6.3";
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs =
+    [glfw]
+    ++ lib.optionals stdenv.isDarwin [darwin.apple_sdk.frameworks.Carbon
+                                      darwin.apple_sdk.frameworks.AppKit
+                                      freetype ]
+    ++ lib.optionals stdenv.isLinux [gtk2 tbb];
+
+  CFLAGS = lib.optionalString stdenv.isLinux "-ltbb"
+             + lib.optionalString stdenv.cc.isClang "-faligned-allocation";
```

Comment:
This will be invalid if both isLinux and isClang is true because of a missing space.